actions-sms.php

```diff
@@ -241,140 +233,135 @@ function freeBikes($number)
 
 function log_sms($sms_uuid, $sender, $receive_time, $sms_text, $ip)
 {
-    global $dbserver,$dbuser,$dbpassword,$dbname;
```

Comment:
yes, this was used before only to connect to the master db, obsoleted by redbeanphp